CMakeLists.txt

```diff
@@ -1,46 +1,88 @@
-cmake_minimum_required (VERSION 2.8)
-project (openmapper)
+cmake_minimum_required (VERSION 3.1)
 
-if("${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION}" LESS 3.1)
-	# Use C++11 : cmake <=3.1
-	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
-else()
-	# Use C++11 : cmake >= 3.1
-	set(CMAKE_CXX_STANDARD 11)
-	set(CMAKE_CXX_STANDARD_REQUIRED ON)
-endif()
+################################
+# OpenMapper
+################################
 
-include_directories(include)
+PROJECT(OpenMapper)
+SET(OM_WRAPPER_LIB "OpenMapper")
+SET(OM_WRAPPER_APP "OpenMapperApp")
 
-##############
-# External Library
-##############
+SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
```

Comment:
for cmake version >=3.1 the recommended flag is: 
set(CMAKE_CXX_STANDARD 11) 
set(CMAKE_CXX_STANDARD_REQUIRED ON) 

ref: 
https://stackoverflow.com/questions/37621342/cmake-will-not-compile-to-c-11-standard
https://cmake.org/cmake/help/v3.1/manual/cmake-compile-features.7.html